### PR TITLE
#100 created nav element

### PR DESCRIPTION
**Pull request by ajoborgvold**
Changed the header design a bit to achieve a nice looking nav bar. Theme switch should be working for the header/nav.
The element is not yet responsive.

### CODE REVIEW

File: assets/css/index.css [hunk 1]
```diff
@@ -8,14 +8,38 @@
...
+.nav-item:hover,
+.nav-item:focus,
+.nav-item:active {
+  text-decoration: underline;
+  box-shadow: unset;
 }
 
+/* Margin and padding decreased to create header layout that works for the nav bar as well */
 .theme-button {
-  float: right;
```
Comment (michaella23) [inline assets/css/index.css:16]: Thank you for commenting these lines out and leaving a note so other contributors can see how new layout is achieved.

File: index.html [hunk 2]
```diff
@@ -14,6 +14,10 @@
 
 <body>
     <header class="header">
+        <nav class="nav-bar">
```
Comment (michaella23) [inline index.html:17]: This works! Thanks for taking on this issue @ajoborgvold!

**Review by michaella23 — APPROVED** [review]
I think these changes look good. Thanks for considering details like the theme switch.

**Comment by michaella23** [comment]
If you'd like to create a new issue to make `nav` responsive, go ahead!

**Review by will-s-205 — COMMENTED** [review]
LGTM

**Pull request merged**